backend/src/controllers/encounter.controller.ts

```diff
@@ -142,6 +142,44 @@ export const getEncounter = async (
   }
 };
 
+export const deleteEncounters = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+  ): Promise<void> => {
+    logger.info("DELETE /encounters/:encounterID request from frontend");
+    const auth_id = req.headers.authorization?.["user_id"];
+    
+    const current_user = await userService.getUserByAuthId(auth_id);
+    const id = req.params.id;
+    const user_encounters = current_user?.encounters;
+    let string_encounters = user_encounters?.map(x => x.toString());
+
+    if (string_encounters?.includes(id.toString())) {
+      try {
+        // Delete encounter from Encounter and Person collection
+        const deleteEncounterResult = await encounterService.deleteEncounter(req.params.id);
+        const deletePersonEncountersResult = await personService.deletePersonEncounters(req.params.id);
+        const deleteUserEncounterResult = await userService.deleteUserEncounter(req.params.id);
+
+        // Notify frontend that the operation was successful
+        if (deleteEncounterResult && deletePersonEncountersResult && deleteUserEncounterResult) {
+          res.sendStatus(httpStatus.OK).end();
```

Comment:
Potentially `204 NO CONTENT` for successful delete operation?

Reply:
Fixed